python/ray/dashboard/modules/job/job_manager.py

```diff
@@ -505,6 +506,8 @@ async def submit_job(
                 "Please use a different submission_id."
             )
 
+        driver_logger = self._get_job_driver_logger(submission_id)
+        driver_logger.info("Runtime env is setting up.")
```

Comment:
We can setup the logger early, but i think the `driver_logger.info("Runtime env is setting up.")` line should moved back right before supervisor was constructed as that's where the runtime env is starting to get setting up.

Reply:
Make sense. Reverted the change.